official/mnist/mnist.py

```diff
@@ -95,7 +107,7 @@ def model_fn(features, labels, mode, params):
             'classify': tf.estimator.export.PredictOutput(predictions)
         })
   if mode == tf.estimator.ModeKeys.TRAIN:
-    optimizer = tf.train.AdamOptimizer(learning_rate=1e-4)
+    optimizer = optimizer_fn(learning_rate=LEARNING_RATE)
```

Comment:
Why the change to a named constant?
The value is used in precisely one place (here), so inlining it avoids indirection when reading it.

If we want to parameterize the learning rate, then perhaps we should do so with a flag and a value in the `params` dictionary. That way everything that is "configurable" is configurable via flags rather than some things by flags and some by named constants in the source file.

Reply:
Another stylistic preference-- buried Magic Numbers can be hard to find and debug, and are prone to copy and paste errors, so I try to avoid them, but can swap back in this case.